Menu/Menu.js

```diff
@@ -33,3 +33,27 @@ let menuItems = [
   Step 6: add the menu component to the DOM.
   
 */
+
+function createMenu(items) {
+  const menuDiv = document.createElement('div');
+  menuDiv.className = "menu";
+
+  const list = document.createElement('ul');
+  menuDiv.appendChild(list);
+  items.forEach((item, i) => {
+    const link = document.createElement('li');
```

Comment:
Just to clarify a possible misconception `li` does not stand fro 'link', it stands for 'list item'